### ISSUE TITLE
eb scale on single-instance environment

### ISSUE THREAD

**Issue by faculezcano** [aws/aws-elastic-beanstalk-cli#64]
### Description

Im currently using a testing environment and i have a scheduled scaling, so eventualy it has 0 instances. My problem is when i try to manually set it to 1 with `eb scale 1` it asks:

> The environment is currently a single-instance. Do you want to change to a load-balancing environment?
WARNING: If you choose yes, the environment and your application will be temporarily unavailable.
(Y/n):

and the when choosing `no` nothing happens.

### Steps to reproduce
 
run `eb scale 1` or `eb scale 0` on a single-instance environment

### Observed result

Scale doesnt apply

### Expected result

Scale applied succesfully

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Amazon Linux 2
2. EBCLI version: 3.17.1

**Comment by elgohr** [comment]
Just out of curiosity, why don't you terminate the environment and start it up fresh?

**Comment by faculezcano** [comment]
> Just out of curiosity, why don't you terminate the environment and start it up fresh?

It seems overkill i just want a simple command to initiate a instance inside the environment.

**Comment by Palakpatel67** [comment]
@faculezcano thanks for reaching out. `eb scale 0` on single instance environment seems to be working fine for me. A single instance environment doesn’t have a load balancer, when you scale up your environment, elastic beanstalk will add new instances and change your environment to a load balanced environment. So when you run  `eb scale 1` on single instance environment, Elastic Beanstalk will update your environment from single instance to load balanced environment.

**Comment by Palakpatel67** [comment]
Closing this issue as there has been no update for an extended period of time.

**Comment by refre5h** [comment]
Sorry for commenting on a closed issue, but I faced this exact issue today. I'd like to manually scale the instance count to 1 for some unexpected work that requires the server to be up, while keeping the environment "single instance" which was the configuration prior to the Time-based Scaling scaled the instance count to 0.

As this issue says, using `eb scale 1` won't work as it'd add a load balancer, which is something I don't want. I've worked around the issue by adding a one-time "scheduled action" to set the desired instance count to 1 in a few minutes.

But the thing is, the Time-based Scaling will maintain the environment's "single instance" status. Is it using a different method than `eb scale 1` for adding an instance? If so, what is it, and it is available as a EB CLI command? I've gone thru the [command reference](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb3-cmd-commands.html) and couldn't find anything.
